### PR TITLE
added filter feature to the expense tracker app

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
Added Filter feature in the Expense Tracker app

### CODE REVIEW

File: Tauqeer-Ahmed-99/01-expense-tracker/src/components/ExpenseFilter.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          <label>Filter by year</label>
+          <select onChange={selectionHandler}>
+            <option value="All">All</option>
+            <option value="2023">2023</option>
+            <option value="2022">2022</option>
+            <option value="2021">2021</option>
+            <option value="2020">2020</option>
+            <option value="2019">2019</option>
+            <option value="2018">2018</option>
+            <option value="2017">2017</option>
```
Comment (faran4engg) [inline Tauqeer-Ahmed-99/01-expense-tracker/src/components/ExpenseFilter.js:22]: why repetitive code ? add it in an array and then loop

**Review by faran4engg — APPROVED** [review]
one change only, do it then merge

**Review by AlishaPrasad — APPROVED** [review]

File: Tauqeer-Ahmed-99/01-expense-tracker/src/components/ExpenseFilter.js (hunk 1 above)
Comment (faran4engg) [reply Tauqeer-Ahmed-99/01-expense-tracker/src/components/ExpenseFilter.js:22]: @Tauqeer-Ahmed-99  ??

**Review by anask601 — APPROVED** [review]

**Review by faran4engg — COMMENTED** [review]
its already approved. Why this hasn't been mergef ? @Tauqeer-Ahmed-99

**Pull request merged**